client/app/styles.css

```diff
@@ -0,0 +1,39 @@
+.color-picker .react-colorful {
+  position: relative;
+  width: auto;
+  height: 75vh;
+}
+
+.Modal {
+  display: flex;
+  top: 550px;
+  left: 40px;
+  right: 40px;
+  bottom: 40px;
+  background-color: papayawhip;
+}
+
+.ReactModal__Overlay {
+  background-color: rgba(255, 255, 255, 0.2) !important;
+}
+
+.ReactModal__Content {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: end;
+  background: rgb(255, 255, 255) !important;
+  border-radius: 10px !important;
+}
+
+.color-picker {
```

Comment:
nit: looks like `color-picker` defined twice, can you also throw this & `close-button` into a style's object in ColorPickerModal.tsx to match other component files?